### PR TITLE
Add users pagination

### PR DESCRIPTION
**Pull request by francocorreasosa**
Solution for #60

### CODE REVIEW

File: client/components/Users/UserPaginator.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+  render() {
+    const { onPageChange, nextPage, pages } = this.props;
+
+    return (
+      <nav className="pull-right">
+        <ul className="pagination pull-right">
+          <li
+            style={{ cursor: 'pointer' }}
+            onClick={() => onPageChange(nextPage - 1)}
```
Comment (sandrinodimattia) [inline client/components/Users/UserPaginator.jsx]: Don't create inline functions:

Create a new function in the class: `onPreviousPage = () => ...`

File: client/components/Users/UserPaginator.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            </a>
+          </li>
+          {[ ...Array(pages) ].map((item, i) => {
+            const page = i + 1;
+            return (
+              <li
+                key={i}
+                style={{ cursor: 'pointer' }}
+                className={nextPage === page ? 'active' : ''}
+                onClick={() => onPageChange(page)}
```
Comment (sandrinodimattia) [inline client/components/Users/UserPaginator.jsx]: Don't create inline functions (bad for perf)

Create a new function in the class: `onNextPage = () => ...`

File: client/components/Users/UserPaginator.jsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+              >
+                <a>{page}</a>
+              </li>
+            );
+          })}
+          <li
+            style={{ cursor: 'pointer' }}
+            onClick={() => onPageChange(nextPage + 1)}
```
Comment (sandrinodimattia) [inline client/components/Users/UserPaginator.jsx]: Same

File: client/containers/Users/Users.jsx [hunk 4]
```diff
@@ -40,6 +40,11 @@ class Users extends Component {
     this.props.fetchConnections();
   };
 
+  onPageChange = (page) => {
+    console.log('page change requested', page);
```
Comment (sandrinodimattia) [inline client/containers/Users/Users.jsx]: Remove this

**Comment by francocorreasosa** [comment]
All requested changes has been made.

**Pull request merged**